src/php/Command/Runtime/RuntimeCommand.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\Command\Runtime;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class RuntimeCommand extends Command
+{
+    private const COMMAND_NAME = 'runtime';
+    private const PHEL_CONFIG_FILE_NAME = 'phel-config.php';
+
+    private string $applicationRootDir;
+
+    private ConfigNormalizer $configNormalizer;
+
+    private RuntimeFileGenerator $runtimeFileGenerator;
+
+    public function __construct(
+        string $applicationRootDir,
+        ConfigNormalizer $configNormalizer,
+        RuntimeFileGenerator $runtimeFileGenerator
+    ) {
+        parent::__construct(self::COMMAND_NAME);
+
+        $this->applicationRootDir = $applicationRootDir;
+        $this->configNormalizer = $configNormalizer;
+        $this->runtimeFileGenerator = $runtimeFileGenerator;
+    }
+
+    protected function configure(): void
+    {
+        $this->setDescription('Generates the PhelRuntime file in vendor');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        /** @var array<string, list<string>> $loaderConfig [ns => [path1, path2, ...]] */
+        $loaderConfig = [];
+        $loaderConfig = $this->loadRootConfig($loaderConfig);
+        $loaderConfig = $this->loadVendorConfig($loaderConfig);
+
+        file_put_contents(
+            $this->applicationRootDir . 'vendor/PhelRuntime.php',
+            $this->runtimeFileGenerator->generate($loaderConfig)
+        );
+
+        $output->writeln('<info>PhelRuntime created/updated successfully!</info>');
+
+        return self::SUCCESS;
+    }
+
+    /**
+     * @return array<string, list<string>>
+     */
+    private function loadRootConfig(array $result): array
+    {
+        $rootPhelConfigPath = $this->applicationRootDir . self::PHEL_CONFIG_FILE_NAME;
+
+        if (is_file($rootPhelConfigPath)) {
+            $pathPrefix = '/..';
```

Comment:
Can you explain this to me. Why this value?

Reply:
I took that value from here: https://github.com/phel-lang/phel-composer-plugin/blob/master/src/DumpRuntime.php#L63 when `isRootPackage === true`

```php
$pathPrefix = $isRootPackage ? '/..' : '/' . $package->getName();
```